account_invoice_analytic_search/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014-2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```

Comment:
IMHO this is reserved to authors. Contributors who migrate should be in the contributors section in the readme.